addOns/automation/src/test/java/org/zaproxy/addon/automation/jobs/JobUtilsUnitTest.java

```diff
@@ -198,6 +201,17 @@ void shouldApplyObjectToType() {
         Object enumBBB = JobUtils.objectToType("bbB", enumeration.class);
         Object map = JobUtils.objectToType(hmap, Map.class);
 
+        Class stringArrayClass = Class.forName("[L" + String.class.getName() + ";");
```

Comment:
Could do `Class<String[]> stringArrayClass = String[].class;`, and the `rawtypes` suppression would no longer be needed.